sklearn/metrics/pairwise.py

```diff
@@ -457,10 +431,13 @@ def pairwise_distances_argmin(X, Y, axis=1, metric="euclidean",
     sklearn.metrics.pairwise_distances
     sklearn.metrics.pairwise_distances_argmin_min
     """
+    if batch_size is not None:
```

Comment:
Just forward the parameters as provided onto `argmin_min` and let it do the warning and conversions.